docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+db:
+  image: mysql:5.6
+  environment:
+    MYSQL_ALLOW_EMPTY_PASSWORD: yes
+    MYSQL_DATABASE: ralph_ng
+    MYSQL_ROOT_PASSWORD:
+    MYSQL_USER: ralph_ng
+    MYSQL_PASSWORD: ralph_ng
+
+web:
+  build: .
+  restart: always
+  environment:
+    - DJANGO_SETTINGS_MODULE=ralph.settings.settings_local
+  ports:
+    - "8000:8000"
+  volumes:
+    - docker/settings_local.py:/opt/ralph/src/ralph/settings/settings_local.py
```

Comment:
I've added a comment that it's only temporary until ralph will handle local settings in other (better) way.